src/js/HRModules/showForms.js

```diff
@@ -0,0 +1,183 @@
+        $id('showForms-list').style.display = 'none';
+        $id('showForms-form').style.display = 'block';
+
+        $id('showForms-form-title').innerHTML = which.title;
+        $id('showForms-form-content').innerHTML = '';
+
+        for (const question of which.questions) {
+            if (question.type.toLowerCase() === 'o') {
+                $id('showForms-form-content')
+                    .appendChild(this.createOpenQuestion(question.number, question.content));
+            } else if (question.type.toLowerCase() === 'w') {
+                $id('showForms-form-content')
+                    .appendChild(
+                        this.createClosedQuestion(question.number,
+                            question.content,
+                            question.answers)
+                    );
+            } else if (question.type.toLowerCase() === 'l') {
+                $id('showForms-form-content')
+                    .appendChild(this.createNumberQuestion(question.number, question.content));
+            }
+        }
+    },
+
+    createOpenQuestion(number, question) {
+        const div = document.createElement('div');
+        div.classList.add('showForms-form-question');
+        div.classList.add('showForms-form-oQuestion');
+
+        const p = document.createElement('p');
+        p.innerHTML = `${number}. ${question}`;
+        div.appendChild(p);
+
+        const input = document.createElement('input');
+        input.placeholder = 'Your answer';
+        div.appendChild(input);
+
+        return div;
+    },
+
+    createClosedQuestion(number, question, answers) {
+        const div = document.createElement('div');
+        div.classList.add('showForms-form-question');
+        div.classList.add('showForms-form-wQuestion');
+
+        const p = document.createElement('p');
+        p.innerHTML = `${number}. ${question}`;
+        div.appendChild(p);
+
+        const commonName = `showForms-form-question-${number}`;
+        for (const [it, answer] of answers.entries()) {
+            const label = document.createElement('label');
+            label.for = `showForms-form-question-${it}`;
+
+            const input = document.createElement('input');
+            input.id = `showForms-form-question-${it}`;
+            input.name = commonName;
+            input.type = 'radio';
+            label.appendChild(input);
+
+            div.appendChild(label);
+
+            const child = document.createElement('div');
+            const dot = document.createElement('div');
+            child.appendChild(dot);
+            label.appendChild(child);
+
+            const span = document.createElement('span');
+            span.innerHTML = answer;
+            label.appendChild(span);
+        }
+
+        return div;
+    },
+
+    createNumberQuestion(number, question) {
+        const div = document.createElement('div');
+        div.classList.add('showForms-form-question');
+        div.classList.add('showForms-form-lQuestion');
```

Comment:
Jak powyżej

Reply:
To co ma być w reszcie?
o - open
w - ?
l - ?

Reply:
Powinno być
o- open
c- closed
l- literal

Reply:
czyli zmieniamy nazewnictwo?

Reply:
Może byc numerical, i tak będzie lepiej niż wcześniej